flixel/input/FlxPointer.hx

```diff
@@ -15,6 +15,7 @@ class FlxPointer
 	
 	private var _globalScreenX:Int = 0;
 	private var _globalScreenY:Int = 0;
+	private static var _cachedPoint:FlxPoint = new FlxPoint();
```

Comment:
I was specifically talking about this `FlxPoint` var here. I agree that the floats would be <strike>quite</strike> even more confusing without the prefix.